config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :offers
   get 'travel_time/get'
```

Comment:
For JSON API, response headers should be `application/vnd.api+json` not `application/json` - don't think this is super urgent (not sure where it should be used in the front end).